File: app/src/main/java/com/example/androidkpnew/MainActivity.java
package com.example.androidkpnew;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;
import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.FirebaseDatabase;

import java.util.HashMap;
import java.util.Map;


public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final EditText satu = findViewById(R.id.satu);
        final EditText dua = findViewById(R.id.dua);
        Button gas = findViewById(R.id.gas);

        DAOEKaryawan jir = new DAOEKaryawan();
        gas.setOnClickListener(view -> {
            Karyawan orang = new Karyawan(satu.getText().toString(),dua.getText().toString());
            jir.add(orang).addOnSuccessListener(suc -> {
                Toast.makeText(this,"Telah Terinput", Toast.LENGTH_LONG).show();
            }).addOnFailureListener(er -> {
                Toast.makeText(this,""+er.getMessage(), Toast.LENGTH_LONG).show();
            });
        });
    }
}

File: app/src/main/java/com/example/androidkpnew/MainActivity.kt
package com.example.androidkpnew;

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import com.example.androidkpnew.R
import com.example.androidkpnew.HomeActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        supportActionBar?.hide()

        Handler().postDelayed({
            val intent = Intent(this@MainActivity, HomeActivity::class.java)
            startActivity(intent)
            finish()
        }, 3000)
    }
}